xbmc/addons/addoninfo/AddonInfoBuilder.cpp

```diff
@@ -161,8 +156,8 @@ bool CAddonInfoBuilder::ParseXML(const AddonInfoPtr& addon, const TiXmlElement*
   const TiXmlElement* backwards = element->FirstChildElement("backwards-compatibility");
   if (backwards)
   {
-    cstring = backwards->Attribute("abi");
-    addon->m_minversion = AddonVersion(cstring ? cstring : "");
+    const std::string& minVersion = StringUtils::StringOrEmpty(backwards->Attribute("abi"));
```

Comment:
I know compilers are quite clever these days, but as the new stringutils function returns an instance, not a ref, I would make that explicit by assigning the return value to a const std::string instance, not a const ref. The compiler will take care about return type optimization to avoid copies.

Reply:
makes sense, applied your suggestions with the latest push